eng/pipelines/libraries/execute-linker-tests-steps.yml

```diff
@@ -0,0 +1,4 @@
+  steps:
+    # Execute tests
+    - script: $(Build.SourcesDirectory)$(dir)build$(scriptExt) -ci -arch x64 $(_osParameter) -s libs.tests -c $(_BuildConfig) /p:LinkerTests=true $(_officialBuildParameter) $(_crossBuildPropertyArg) /bl:$(Build.SourcesDirectory)/artifacts/log/$(buildConfigUpper)/LinkerTests.binlog
```

Comment:
We flow archType to extraSteps in `global-build-job` so use that instead. I know we just want to run them on `x64` but that should be driven by `platform-matrix.yml`.